src/content/cloud/external-resources.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: External Resources
```

Comment:
I am missing a comment about the common use case where:
- External resource is deployed in your `deploy` commands
- External resource is destroyed in your `destroy` commands
- You can configure the `deploy/destroy` image
- External resources are deleted when the namespace is deleted. This in combination with the Okteto garbage collector makes sure you never leave cloud resources hanginf for ever.

Reply:
@pchico83 Do we have a sample `deploy` or `destroy` config we could include here?

Or is the broader point your one from our call today where you're saying the external resources can be included as part of a new image and that image can be configured in the `deploy` and `destroy` sections?

Reply:
@omnomagonz external resources let you manage the life cycle of cloud external resources as part of your development environment. In addition to the option to customize endpoints and notes.txt in the UI, you can configure your own container image to run your deploy/destroy commands the same way for all your developers.
@rberrelleza is working on samples. @rberrelleza could you provide a good sample here?

Reply:
Hopefully I addressed these gaps in the latest commit 🤞